### PR TITLE
Move .set/.add methods from SQLExecutor.Builder to SQLExecutor

### PR DESCRIPTION
**Pull request by mfussenegger**
See commits

### CODE REVIEW

File: server/src/testFixtures/java/io/crate/testing/SQLExecutor.java [hunk 1]
```diff
@@ -248,51 +255,91 @@ public PlannerContext getPlannerContext(ClusterState clusterState, Random random
...
+            ClusterState.builder(prevState).nodes(builder).build());
+    }
+
+    /**
+        * Publish the current {@link ClusterState} so new instances of
+        * {@link org.elasticsearch.cluster.ClusterStateListener} like e.g. {@link Schemas} will consume it and
+        * build current schema/table infos.
+        *
+        * The {@link Metadata#version()} must be increased to trigger a {@link ClusterChangedEvent#metadataChanged()}.
+        */
```
Comment (seut) [inline server/src/testFixtures/java/io/crate/testing/SQLExecutor.java:279]: Wrong indention?

File: server/src/testFixtures/java/io/crate/testing/SQLExecutor.java [hunk 2]
```diff
@@ -1046,4 +692,357 @@ public Session createSession() {
...
+        ClusterServiceUtils.setState(clusterService, allocationService.reroute(updatedState, "assign shards"));
+        return this;
+    }
+
+
+    /**
+        * Add a view definition to the metadata.
+        * Note that this by-passes the analyzer step and directly operates on the clusterState. So there is no
+        * resolve logic for columns (`*` is not resolved to the column names)
+        */
```
Comment (seut) [inline server/src/testFixtures/java/io/crate/testing/SQLExecutor.java:885]: Wrong indention?

**Review by seut — APPROVED** [review]
👍 spotted only 2 wrong javadoc indention.

**Comment by mfussenegger** [comment]
> 

Thanks, must have messed that up during the last rebase.

**Pull request merged**